sonar-batch/src/main/java/org/sonar/batch/report/ReportPublisher.java

```diff
@@ -19,9 +19,12 @@
  */
 package org.sonar.batch.report;
 
+import org.sonar.batch.analysis.AnalysisProperties;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.batch.analysis.AnalysisProperties'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)